src/flair/flair_spliceevents.py

```diff
@@ -0,0 +1,483 @@
+#! /usr/bin/env python3
```

Comment:
Is this a new flair module being added?

for new or heavily rewritten modules, please add to flake8.mk and run make flake8 to bring into conformancs